src/backend/InvenTree/company/serializers.py

```diff
@@ -399,12 +391,7 @@ def __init__(self, *args, **kwargs):
         # Check if 'available' quantity was supplied
         self.has_available_quantity = 'available' in kwargs.get('data', {})
 
-        # TODO INVE-T1 support complex filters
         brief = kwargs.pop('brief', False)
-        detail_default = not brief
-        part_detail = kwargs.pop('part_detail', detail_default)
-        supplier_detail = kwargs.pop('supplier_detail', detail_default)
-        manufacturer_detail = kwargs.pop('manufacturer_detail', detail_default)
```

Comment:
This is a breaking change in the API, is that intended?

Reply:
The default behaviour should be the same, but the API schema will need to be updated as these are now filterable